src/server/zset_family.cc

```diff
@@ -2599,6 +2636,296 @@ void ZSetFamily::GeoDist(CmdArgList args, ConnectionContext* cntx) {
                              distance_multiplier);
 }
 
+// Search all eight neighbors + self geohash box
+int membersOfAllNeighbors(ConnectionContext* cntx, string_view key, const GeoHashRadius* n,
```

Comment:
Do you use `n`? It's hard to ctrl-f, but I couldn't find where it is used

Reply:
neighbors[0] = n->hash;
  neighbors[1] = n->neighbors.north;
  neighbors[2] = n->neighbors.south;
  neighbors[3] = n->neighbors.east;
  neighbors[4] = n->neighbors.west;
  neighbors[5] = n->neighbors.north_east;
  neighbors[6] = n->neighbors.north_west;
  neighbors[7] = n->neighbors.south_east;
  neighbors[8] = n->neighbors.south_west;

Reply:
Indeed, I missed that. Sorry!
Consider renaming this variable though